### PR TITLE
Update data loss section with new links and fix typos

### LINKED ISSUE #446: Update Data loss - prevention and recovery section
mostly updating some broken or obsolete links, as well as fix some typos

### PR DESCRIPTION
**Pull request by Hooverdan96**
Fixes #446.

### This pull request's proposal
 Changes to be committed:
	modified:   data_loss.rst
- updated broken link
- updated obsolete links from btrfs wiki to readthedocs version
- minor typo corrections

### Checklist
- [X] With the proposed changes no Sphinx errors or warnings are generated.
- [X] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Comment by Hooverdan96** [comment]
I am not entirely sure what that error means during the link check ...on the sphinx simulation on my system all seems fine.

File: data_loss.rst [hunk 1]
```diff
@@ -109,10 +110,10 @@ The core mechanisms used to counter corruption are:
 
 .. _dlthread_monitoring:
 
-Web-UI and Data-Integrity threat monitoring
+Web-UI and Data Integrity threat monitoring
 -------------------------------------------
 
-The modern Rocksor Web-UI is able to indicates btrfs 'device missing', and btrfs sensed errors;
+The modern Rocktsor Web-UI is able to indicates btrfs 'device missing', and btrfs sensed errors;
```
Comment (FroggyFlox) [inline data_loss.rst:116]: Just a typo.
```suggestion
The modern Rockstor Web-UI is able to indicate btrfs 'device missing', and btrfs sensed errors;
```

**Comment by Hooverdan96** [comment]
thanks @FroggyFlox, considering that I went in to fix some typos and then generated new ones is pretty embarrassing :)

**Comment by Hooverdan96** [comment]
For simplicity I squashed the multiple commits into 1.

**Comment by phillxnet** [comment]
@Hooverdan96 & @FroggyFlox Thanks for all the work here - nice to have this super important section get some more attention: as always much appreciated. Nice improvements.

And my apologies (again) for yet more 'it's' errors on my part most likely.

**Comment by phillxnet** [comment]
PR product PRODUCTION published.

**Pull request merged**